docs/uvcal_tutorial.rst

```diff
@@ -42,6 +42,23 @@ gives the beginning and end of the time range. The local sidereal times follow a
 pattern, UVCal objects should have either an ``lst_array`` or an ``lst_range`` attribute
 set.
 
+Generating calibration solutions typically requires choosing a convention concerning how
+polarized sky emission is mapped to the linear polarizations of the instrument. For
```

Comment:
I prefer "is mapped to instrumental polarizations." Because telescopes with right and left circular polarization feeds have exactly this same issue of averaging vs summing.

I think it's fine to leave the following example with linear pols, I just don't want to cast this as limited to linear pol.

Reply:
Got it, good point.